core-requirements.txt

```diff
@@ -11,7 +11,7 @@ psutil>=5.6.3
 requirements-parser>=0.2.0
 shap>=0.36.0
 texttable>=1.6.2
-woodwork==0.0.11
+woodwork==0.3.1
```

Comment:
nit: can we set to `>=0.3.1`?

Reply:
This was restricted to specific Woodwork version because early on there was breaking changes. 
We have been more proactive noting the breaking changes in Woodwork in the release notes, and incrementing the version properly.
- https://woodwork.alteryx.com/en/latest/release_notes.html#breaking-changes